src/components/ContextMenu.vue

```diff
@@ -1,12 +1,31 @@
 <script lang="ts">
 import store from '@/store';
-import { computed } from '@vue/composition-api';
+import { computed, ref, watchEffect } from '@vue/composition-api';
 
 export default {
   setup() {
     const rightClickMenu = computed(() => store.getters.rightClickMenu);
     const selectedNodes = computed(() => store.getters.selectedNodes);
     const network = computed(() => store.getters.network);
+    const numericVariables = computed(() => {
+      const numericColumns = Object.entries(store.getters.columnTypes)
+        .map(([key, value]) => (value === 'number' ? key : ''));
+
+      const numericColumnsSet = new Set(numericColumns);
+      numericColumnsSet.delete('');
```

Comment:
I think this is a classic use case for `.filter()` over `.map()`.

Reply:
Fixed with 4d9fcef. Guess I got tunnel vision on this code block 😂